Runtime/Scripts/DefaultMaterialGenerator.cs

```diff
@@ -29,7 +31,31 @@ public class DefaultMaterialGenerator : IMaterialGenerator {
             material.mainTextureScale = TEXTURE_SCALE;
             material.mainTextureOffset = TEXTURE_OFFSET;
 
-            if(gltfMaterial.pbrMetallicRoughness!=null) {
+            //added support for KHR_materials_pbrSpecularGlossiness
+            if (gltfMaterial.extensions != null) {
+                Schema.PbrSpecularGlossiness specGloss = gltfMaterial.extensions.KHR_materials_pbrSpecularGlossiness;
+                if (specGloss != null) {
+                    material.shader = Shader.Find("Standard (Specular setup)");
+                    var diffuseTexture = GetTexture(specGloss.diffuseTexture, textures, images);
+                    if (diffuseTexture != null) {
+                        material.SetTexture("_MainTex", diffuseTexture);
```

Comment:
I try to avoid string literal based material Set* methods in favour of property ID based ones.
in this case `material.mainTexture =` is preferable.